qiwis.py

```diff
@@ -171,6 +178,7 @@ def load(self, appInfos: Mapping[str, AppInfo]):
         """
         for name, info in appInfos.items():
             self.createApp(name, info)
+        logger.info("Loaded %d app(s)", len(appInfos))

```

Comment:
For other uses, I also prefer the 'f-string'!
However, the `logging` system does some [optimization](https://docs.python.org/3.7/howto/logging.html#optimization):
> Formatting of message arguments is deferred until it cannot be avoided.

If we use the f-string, it will be directly converted to a string, even if the message is not printed at all.

See also: [Logging Cookbook](https://docs.python.org/3.7/howto/logging-cookbook.html#using-particular-formatting-styles-throughout-your-application).
However, this is not

Reply:
Wow, I never thought it would be for this reason!
Now, never mind😃